### DISCUSSION TITLE
Netbeans autocomplete

### DISCUSSION THREAD

**Discussion by zukester** [apache/netbeans#7025] [General]
Product Version: Apache NetBeans IDE 20
Java: 11.0.21; OpenJDK 64-Bit Server VM 11.0.21+1
Runtime: OpenJDK Runtime Environment 11.0.21+1
System: Linux version 5.15.0-92-generic running on amd64; UTF-8; en_US (nb)
User directory: /home/zukestermint/.netbeans/20
Cache directory: /home/zukestermint/.cache/netbeans/20

In the php editor, when I type <br> it auto-completes with <br>

I have looked in two search engines, Youtube, online on the apache site, and here, as well as Stackexchange.  No relevant help.irrelevant results.   I am in Linux Mint latest version, checked multiple times per day, updated on everything multiple times per day, and so is Netbeans, according to Netbeans.

How do I take the  out of there?  I have searched for and opened many module files, no idea where to even find it, and searched online for it.

Thanks,
Zukestermint

**Comment by matthiasblaesing** [comment]
Please read your post before you post it. You post in its current form makes no sense.

**Comment by zukester** [comment]
I did read it.  What is so hard to understand?  Go in the editor, type <br> and hit enter.  That is how autocomplete works.

<br>  every time.

No one needs a comment on a line break.  I understand that there is a way to change that to remove the . I want to know where and how is the question.  It is a logical question.

**Comment by matthiasblaesing** [comment]
This is what you posted:

![image](https://github.com/apache/netbeans/assets/2179736/b7f40579-564d-4685-afea-0f70a5c1637b)

I assume the whitespace (red markers) should be filled with something. It is not.

**Reply by zukester** [reply]
No I left the white spaces there.  It is called emphasis
ifyouwantmetonothaveasinglewhitespacethenIcandothatalsoIneedhelp.

**Reply by mbien** [reply]
this isn't emphasis its not quoting code segments properly. Your post does not render correctly and is therefore not readable.

Making fun of people who want to help you is also a bad strategy if you ask for help - good luck with that.

**Comment by Chris2011** [comment]
hey @zukester sry but I also don't get it. What is your problem what you want to describe? What is not working?

**Comment by zukester** [comment]
I do not remember now, but it was typing < for many, many things in the html and it always put "<a href  " in there, and the autocomplete had nothing to do with what I was typing.  I fixed it somehow.
I figured it out, eventually.  Wish I could remember how.  Thanks
